buildscripts/upgrade-082090/Makefile.mk

```diff
@@ -0,0 +1,43 @@
+# NOTE: The Upgrade design is changing and this file will be deprecated 
+# in the future. Currently the upgrade process used by this only applies
+# to 0.8.2 to 0.9.0. When the new implementation is updated to work for
+# all upgrade paths including this one, this file will be removed. 
+
+# This binay is not built as part of the CI, if there is an issue, 
+# the image needs to be built and pushed manually. The image-tag
+# needs to specifiy this is for 082090. 
+
+# Specify the name for the binaries
+UPGRADE-082090=upgrade-082090
+UPGRADE-082090-TAG=082090-${IMAGE_TAG}
+
+# build upgrade binary
+.PHONY: upgrade-082090
+upgrade-082090:
+	@echo "----------------------------"
+	@echo "--> ${UPGRADE-082090}       "
+	@echo "----------------------------"
+	#PNAME is the sub-folder in ./bin where binary will be placed. 
```

Comment:
Why are we printing this comments in execution?

Reply:
good one! fixed.